timers.c

```diff
@@ -576,7 +581,7 @@
         traceENTER_vTimerSetReloadMode( xTimer, xAutoReload );
 
         configASSERT( xTimer );
-        taskENTER_CRITICAL();
+        taskLOCK_DATA_GROUP( &xTaskSpinlock, &xISRSpinlock );
```

Comment:
`xTaskSpinlock` and `xISRSpinlock` only exist when portUSING_GRANULAR_LOCKS is enabled. Although these two variables are ignored in macro expansion for non-granular lock kernel, I would like to suggest we hide this information in macro. For example,
```C
/* Non-granular lock version. Single core and SMP. */
#define taskLOCK_TIMER_DATA_GROUP  taskENTER_CRITICAL

/* Granular lock version. */
#define taskLOCK_TIMER_DATA_GROUP() portLOCK_DATA_GROUP( &xTaskSpinlock, &xISRSpinlock )
```